### PR TITLE
complete js-basics_console-methods

### PR DESCRIPTION
**Pull request by Scot-MacDonald**

### CODE REVIEW

File: JS-basics/js-basics_console-methods/js/index.js [hunk 1]
```diff
@@ -6,16 +6,30 @@ console.clear();
...
 
+console.log(coding);
+
 // 3. Store the sentence 'Happy hacking!' in the variable named 'hacking'.
 
+const hacking = "Happy hacking!";
+
 // 4. Display the variable 'hacking' as a message in the console.
 
+console.log();
```
Comment (kayachakmak) [inline JS-basics/js-basics_console-methods/js/index.js]: you forgot to put your hacking variable inside of the paranthesis.

**Review by kayachakmak — COMMENTED** [review]
needs update. Good luck!

File: JS-basics/js-basics_console-methods/js/index.js (hunk 1 above)
Comment (Scot-MacDonald) [reply JS-basics/js-basics_console-methods/js/index.js]: Thanks, done now!

**Review by kayachakmak — APPROVED** [review]
Congrats!

**Pull request merged**